worlds/raft/__init__.py

```diff
@@ -115,14 +116,14 @@ def create_regions(self):
         create_regions(self.multiworld, self.player)
     
     def get_pre_fill_items(self):
-        if self.multiworld.island_frequency_locations[self.player] in [0, 1, 2, 3]:
+        if self.options.island_frequency_locations.value in [0, 1, 2, 3]:
```

Comment:
```suggestion
        if self.options.island_frequency_locations.value <= 3:
```
If you decide to keep with the magic numbers though, probably better to just use <= instead of this. Would still recommend swapping this to the other method in the previous comment, though.